### ISSUE TITLE
ol/interaction/Draw option stopClick: true is not working

### ISSUE THREAD

**Issue by Ruahonen** [openlayers/openlayers#12147]
Version 6.5.0
ol/interaction/Draw option stopClick: true is firing event ol/Map singleclick.

Singleclick event should not be fired when drawing interaction is on.

**Comment by EvertEt** [comment]
I can replicate this with the following sandbox code: the map will zoom on double click to finish the drawing

```js
import "ol/ol.css";
import Draw from "ol/interaction/Draw";
import Map from "ol/Map";
import View from "ol/View";
import { OSM, Vector as VectorSource } from "ol/source";
import { Tile as TileLayer, Vector as VectorLayer } from "ol/layer";

var raster = new TileLayer({
  source: new OSM()
});

var source = new VectorSource({ wrapX: false });

var vector = new VectorLayer({
  source: source
});

var map = new Map({
  layers: [raster, vector],
  target: "map",
  view: new View({
    center: [-11000000, 4600000],
    zoom: 4
  })
});

var draw = new Draw({
  source: source,
  type: "Polygon",
  stopClick: true
});

draw.on("drawend", () => map.removeInteraction(draw));

map.addInteraction(draw);

```

**Comment by ahocevar** [comment]
@EvertEt You can `npm install ol@dev` to get a package that includes all merged pull requests.

**Comment by EvertEt** [comment]
Thanks @ahocevar , that is indeed what I've done now to verify it is indeed fixed.
It feels a bit more dangerous to install a dev version in a prod environment but we might just need to until the next version is released.
Thanks for the quick reply

**Comment by ahocevar** [comment]
The dev builds go through the same test procedure as the official releases, so you should not need to worry.

**Comment by ahocevar** [comment]
Fixed with #11892
